developments_build/sql/devdb_create_init.sql

```diff
@@ -175,13 +178,26 @@ JOBNUMBER_relevant as (
 	existingheight as Height_Init,
 	proposedheight as Height_Prop,
 	totalconstructionfloorarea as ConstructnSF,
-	horizontalenlrgmt as Enlrg_Horiz,
-	verticalenlrgmt as Enlrg_Vert,
+
```

Comment:
See [issue 54](https://github.com/NYCPlanning/db-developments/issues/54)
Maybe change to:
```sql
(CASE WHEN (horizontalenlrgmt = 'Y' AND verticalenlrgmt <> 'Y') THEN 'Enlrg_Horiz'
	WHEN (horizontalenlrgmt <> 'Y' AND verticalenlrgmt = 'Y') THEN 'Enlrg_Vert'
WHEN (horizontalenlrgmt = 'Y' AND verticalenlrgmt = 'Y') THEN 'Enlrg_Horiz and Enlrg_Vert'
END)  as enlargement
```